dev/com.ibm.ws.install.featureUtility/resources/com/ibm/ws/install/featureUtility/internal/resources/FeatureUtilityToolOptions.nlsprops

```diff
@@ -72,6 +72,17 @@ installFeature.option-desc.--from=\
 \tUse this option to specify the file path of a Maven artifact         \n\
 \trepository as a source for featureUtility to install assets.
 #------------------------------\n at 72 chars -- leading tab-----------\n\#
+installFeature.option-key.--verify=\
```

Comment:
adjust line lengths as needed, I'm not sure what the max length is

Reply:
Followed the same format as the other options.